TasksMeter.c

```
@@ -63,13 +64,52 @@ static void TasksMeter_display(const Object* cast, RichString* out) {
+   int procLen = xSnprintf(tmp, sizeof(tmp), "%d", (int)this->values[2]);
+
+   int thrStart = captionLen + procLen + 2; // +2 for ", "
+   int thrLen   = xSnprintf(tmp, sizeof(tmp), "%d", (int)this->values[1]);
+   int thrEnd   = thrStart + thrLen + 4;    // +4 for " thr"
+
+   int kthrStart = thrEnd + 2;              // +2 for ", "
+   int kthrLen   = xSnprintf(tmp, sizeof(tmp), "%d", (int)this->values[0]);
+   int kthrEnd   = kthrStart + kthrLen + 5; // +5 for " kthr"
+
+   if (relX >= thrStart && relX < thrEnd) {
+      settings->hideUserlandThreads = !settings->hideUserlandThreads;
```

Cppcheck (2.21.0)

[style] The function 'Table_compact' is never used.

(unusedFunction)